src/Routing/Annotations/Annotations/Controller.php

```diff
@@ -34,7 +34,9 @@ public function modifyCollection(EndpointCollection $endpoints, ReflectionClass
     protected function prefixEndpoints(EndpointCollection $endpoints)
     {
         foreach ($endpoints->getAllPaths() as $path) {
-            $path->path = $this->trimPath($this->prefix, $path->path);
+            if(!$path->no_prefix) {
```

Comment:
Can you move the `if` to before the `foreach`? No point in looping if we don't need to.
Also please add a space between the `if` and the `(`

Reply:
Thanks for your answer. The no_prefix is added (or not) in every route, so it's mandatory to check it in each iteration...

Reply:
Space added as you request :)

Reply:
Oh doh, totally misread that check 😂